src/Symfony/Bridge/Twig/Mime/BodyRenderer.php

```diff
@@ -24,19 +27,13 @@ final class BodyRenderer implements BodyRendererInterface
 {
     private Environment $twig;
     private array $context;
-    private HtmlConverter $converter;
+    private HtmlToTextConverterInterface $converter;
 
-    public function __construct(Environment $twig, array $context = [])
+    public function __construct(Environment $twig, array $context = [], HtmlToTextConverterInterface $converter = null)
     {
         $this->twig = $twig;
         $this->context = $context;
-        if (class_exists(HtmlConverter::class)) {
-            $this->converter = new HtmlConverter([
-                'hard_break' => true,
-                'strip_tags' => true,
-                'remove_nodes' => 'head style',
-            ]);
-        }
+        $this->converter = $converter ?: (interface_exists(HtmlConverterInterface::class) ? new LeagueHtmlToMarkdownConverter() : new DefaultHtmlToTextConverter());
```

Comment:
`interface_exists(HtmlConverterInterface::class)` really? Should this check for a League* class?

Reply:
Yes, see `use League\HTMLToMarkdown\HtmlConverterInterface`, that interface IS the one from League ;)